spec/system/admin/organizations_system_spec.rb

```diff
@@ -151,5 +151,30 @@
       expect(page).to have_content("Default email text")
       expect(page).to have_content("Users")
     end
+
+    it "can promote a user to org_admin in the organization" do
```

Comment:
These 2 tests can be request tests - the interaction isn't really important, only whether the buttons display as we expect. System tests are slow and flaky and should be avoided unless we're doing fancy workflow or JavaScript interactions.